tests/run-make/CURRENT_RUSTC_VERSION/rmake.rs

```diff
@@ -10,11 +10,11 @@ use run_make_support::{aux_build, rustc};
 fn main() {
     aux_build().input("stable.rs").emit("metadata").run();
 
-    let mut stable_path = PathBuf::from(env!("TMPDIR"));
-    stable_path.push("libstable.rmeta");
-
-    let output =
-        rustc().input("main.rs").emit("metadata").extern_("stable", &stable_path).command_output();
+    let output = rustc()
+        .input("main.rs")
+        .emit("metadata")
+        .extern_("stable", "libstable.rmeta")
+        .command_output();
```

Comment:
Reminder (for myself): pre-existing; this needs to use `run()` to check exit status.